releasewarrior/config.py

```diff
@@ -80,7 +80,9 @@ def find_repo_path(search_path):
     'cdntest',
     'mirrors',
     'pushapk',
-    'signoff',
     'publish',
+    'signoff',
```

Comment:
I'm afraid we need to revisit all templates and rearrange the steps here too. Will have a look today to rearrange the items.